src/HitStatistics.js

```diff
@@ -39,15 +64,7 @@ export class HitStatistics {
 
   archiveIfNeeded() {
     if (this.hoursPassed() >= this.statisticTtlInHours) {
-      const now = new Date()
```

Comment:
we don't need all this logic, `initForCache` rewrites entire entry to an empty record anyway